### DISCUSSION TITLE
Interactive demos

### DISCUSSION THREAD

**Discussion by dankidigital** [swup/swup#863] [General]
Replit demos/embeds aren't working for us at the moment:

|https://swup.js.org/getting-started/demos/

(This one for example https://replit.com/replid/ff8ac999-d7e2-43b6-ba45-f66b42906365)

Anyone else experiencing the same?

**Comment by daun** [comment]
Looks like Replit [turned off static hosting](https://repl.co/) and requires setting up with their new Deployments offering. We'll need to look into deploying the demos on their new feature. Feel free to open an issue in the [swup docs](https://github.com/swup/docs) repo so we don't lose track of it.
